dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/osgi/webapp/WebApp.java

```diff
@@ -1023,6 +1023,12 @@ protected void scanForHandlesTypesClasses(
                       if (!WCCustomProperties.EXCLUDE_ALL_HANDLED_TYPES_CLASSES) {
                           String actualClassReason = "Selection of handlesType class [ " + handledTypeName + " ]";
                           addClassToHandlesTypesStartupSet(handledTypeName, startupTypes, actualClassReason);
+                      } else {
+                        if ( enableTrace ) {
+                            logger.logp(Level.FINE, CLASS_NAME, methodName,
+                                        "Skipping class, {0}, is not added to the ServletContainerInitializers in the application: {1}.",
```

Comment:
I think it would be good to have a message here, but if any changes should be made to it.

Reply:
Should I say type instead of class?

Reply:
I think the terms are interchangeable here, but `type` might be a little more clear in the context of the method